packages/ketchup/src/f-components/f-text-field/f-text-field.scss

```diff
@@ -495,6 +495,22 @@
         --kup_image_width: 16px;
         width: 16px;
       }
+
```

Comment:
Why the need to duplicate this logic from the cell?

Reply:
Just leftover code, removed it 😌

Reply:
Removing it from f-cell means that only the shapes including a textfield can display the indicator (for example, a progress bar or a chip won't).
Is this the desired expectation?